tfjs-layers/src/layers/nlp/tokenizers_utils.ts

```diff
@@ -242,18 +249,16 @@ export function regexSplit(
   });
 }
 
-export async function tensorToStringArr(input: Tensor): Promise<string[]> {
-  return await input.data() as unknown as string[];
+export function tensorToArr(input: Tensor): unknown[] {
+  return input.dataSync() as unknown as unknown[];
 }
```

Comment:
You might be able to use [`tensor.arraySync()` instead](https://js.tensorflow.org/api/latest/#tf.Tensor.arraySync), although it returns a `number[]`, so maybe not.

Reply:
Sounds good! I've added this to my notes so I can come back to this once the return type of `arraySync` has been updated.